02-oop-inheritance/keypoints/2_1-6/script.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Created by MaximilianoDaniel on 15/09/2015.
+ * Exercises done
+ * 1- Create a Movie object
+ * 2- Instantiate some of your favorite movies and play with them in the console.
+ * 3- Add a MovieObserver class that listen for "playing" and "sttoped" events
+ * 4- Publish "playing" event on Movie.play()
+ * 5- Publish "stopped" event on Movie.stop()
+ * 6- Log to console when each event is fired
+ * */
+
+var inheritPrototype = function (childClass, parentClass){
+
```

Comment:
This line is not needed. No need of change for this exercise, try to be aware of this for next exercise.